packages/extension/src/providers/ethereum/networks/skale/calypso.ts

```diff
@@ -0,0 +1,31 @@
+import { NetworkNames } from "@enkryptcom/types";
+import {
+  SkaleParams,
+  createSkaleEvmNetwork,
+  ICustomSKALEAsset,
+} from "./skale-base";
+
+const skaleOptions: SkaleParams = {
+  name: NetworkNames.SkaleCalypso,
+  name_long: "SKALE | Calypso NFT Hub",
+  chainName: "honorable-steel-rasalhague",
+  chainID: "0x5d456c62",
+  icon: "skaleCalypsoNFTHub.png",
+};
+
+const assets: ICustomSKALEAsset[] = [
+  {
+    name: "Ethereum Clone",
+    symbol: "ETHC",
+    address: "0xD2Aaa00700000000000000000000000000000000",
+    coingeckoID: "ethereum",
+  },
```

Comment:
Remove, this token should never be used

Reply:
Removed - will add it later when we will switch to use the same token instead of wrapped